cmake/modules/FindMUPARSER.cmake

```diff
@@ -44,7 +44,14 @@ IF(EXISTS ${MUPARSER_INCLUDE_DIR}/muParserDef.h)
   FILE(STRINGS "${MUPARSER_INCLUDE_DIR}/muParserDef.h" MUPARSER_VERSION_STRING_LINE
     REGEX "#define MUP_VERSION _T"
     )
- 
+
+  IF("${MUPARSER_VERSION_STRING_LINE}" STREQUAL "")
+    # try again with the newer version format (starting in at least 2.3.2)
+    FILE(STRINGS "${MUPARSER_INCLUDE_DIR}/muParserDef.h" MUPARSER_VERSION_STRING_LINE
+      REGEX "string_type ParserVersion = string_type"
```

Comment:
Would you mind showing in the comment here (and for the old version above) what the expected string looks like?

Reply:
Good idea. Fixed!